src/appleseed.common/application/application.cpp

```diff
@@ -131,6 +131,9 @@ void Application::check_compatibility_with_host(Logger& logger)
 
 namespace
 {
+    static char g_root_path[FOUNDATION_MAX_PATH_LENGTH + 1] = { 0 };
+    static bool g_is_root_path_set = false;
```

Comment:
Why prefixing it with `g_` ?

Reply:
`g_` stands for "global", as in "global variable".